README.md

```diff
@@ -1,70 +1,68 @@
-# Getting Started with Create React App
+# Bookstore CMS Project
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Preview
 
-## Available Scripts
 
-In the project directory, you can run:
+## Features
 
-### `npm start`
+## Project SetUp
+Get a local copy of the file. Clone with HTTPS following the steps below.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### For Windows
+- In your search box, type cmd, click on cmd to open the command prompt.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+### For Linux, Mac OS
+- In your search box, type terminal, click on the terminal to launch it.
 
-### `npm test`
+- Copy and paste the command below in your terminal and press enter from your keyboard.
+```
+git clone https://github.com/Zubenna/my-bookstore-project.git
+```
+## Access Project Folder 
+- Depending on where the project files are downloaded to, either Hard Disk(HDD), Desktop or Documents directory in your computer after completing the step above.
+- Double click on the folder calculator to open it.
+- Inside the folder, right-click and click on terminal or cmd depending on your Operating System to open it.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Set Up Project Locally in Your Computer
+- In the cmd or terminal, type the command below;
+```
+npm install
+```
+- Wait for some time to install all the app packages.
+- Once the installation is completed.
+- Type the command below in your cmd or terminal to start the project locally. Wait for some time.
+```
+npm start
+```
+- Once the server is fully started, you will see the project opening a different tab on your browser at port http://localhost:3000/. The page now displays the calculator app built with react.js. You can use the calculator now.
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Live Version
+Launch live version.
+- [Launch App](https://zubenna-bookstore.herokuapp.com/)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Testing
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Run Test
 
-### `npm run eject`
+## Built With
+- React.js
```

Comment:
I think you can more technologies you're going to use for this project.
kindly do this before merging....
👀  on you....